packages/client/src/raffles/types/raffles.types.ts

```diff
@@ -0,0 +1,49 @@
+export enum RAFFLE_STATUS {
+  DRAFT = 'Draft',
+  READY = 'Ready',
+  ACCEPTION_ENTRIES = 'AcceptingEntries',
```

Comment:
I think there's a typo:
```suggestion
ACCEPTING_ENTRIES = 'AcceptingEntries',
```

Reply:
done